client/data/domain/TipObjectFactory.js

```diff
@@ -64,12 +82,18 @@ tieData.factory('TipObjectFactory', [
     };
 
     /**
-     * Whether the given lines of code trigger the tip.
+     * Whether the given lines of code activate the tip.
      *
      * @param {Array<string>} codeLines The lines of code to examine.
      * @returns {boolean}
      */
-    Tip.prototype.isTriggeredBy = function(codeLines) {
+    Tip.prototype.isActivatedBy = function(codeLines) {
```

Comment:
Probably should add Karma tests for this.

Reply:
Added Karma tests.